### PR TITLE
Automate release version updating

### PR DESCRIPTION
**Pull request by paulb777**
With the required -version parameter, the new script will update all files in
the repo based on the versions in Releases/Manifests/{version}.json.

It will create a release branch, push and tag the updates, and push the
updated podspecs to cpdc-internal.

I verified that the output matches what was done manually in the 5.3.0 release.

### CODE REVIEW

File: Releases/Manifests/5.3.0.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (davidair) [inline Releases/Manifests/5.3.0.json]: Can we add a comment here about what it is, as well as (in a comment) what the version of Firebase is?

File: Releases/update-versions.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""update-versions.py creates a release branch and commit with version updates.
+
+With the required -version parameter, this script will update all files in
```
Comment (davidair) [inline Releases/update-versions.py]: Please make it --version

File: Releases/update-versions.py [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+def SetupArguments():
+  """SetupArguments sets up the set of command-line arguments.
+
+  Returns:
+    Args: The set of command line arguments
+  """
+  parser = argparse.ArgumentParser(description='Update Pod Versions')
+
+  parser.add_argument('-version', required=True, help='Firebase version')
```
Comment (davidair) [inline Releases/update-versions.py]: --version

File: Releases/update-versions.py [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+
+  parser.add_argument(
+      '-test_mode',
```
Comment (davidair) [inline Releases/update-versions.py]: --test_mode

File: Releases/update-versions.py [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+      dest='test_mode',
+      action='store_true',
+      help='Log commands instead of updating public repo')
+
+  parser.add_argument(
+      '-tag_update',
```
Comment (davidair) [inline Releases/update-versions.py]: --tag_update

File: Releases/update-versions.py [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+  """
+  pods = version_data.keys()
+  pods.insert(0, pods.pop(pods.index('FirebaseCore')))  # Core should be first
+  for pod in pods:
+    LogOrRun('pod cache clean {} --all'.format(pod))
+    if pod == 'FirebaseFirestore':
+      warnings_ok = ' --allow-warnings'
+    else:
+      warnings_ok = ''
+    LogOrRun('pod repo push cpdc-internal-spec {}.podspec{}'.format(
```
Comment (davidair) [inline Releases/update-versions.py]: Please do not hardcode

Comment (davidair) [inline Releases/update-versions.py]: Can we convert to json first please?

File: Releases/Manifests/5.3.0.json (hunk 1 above)
Comment (paulb777) [reply Releases/Manifests/5.3.0.json]: Since json doesn't allow comments, I added a README

**Pull request merged**